docker-for-ibm-cloud/administering-swarms.md

```diff
@@ -69,7 +70,7 @@ To create a Docker EE for IBM Cloud cluster from the CLI:
 
    > Tip to set environment variables
    >
-   > You can set your infrastructure API credentials and Docker EE installation URL as environment variables so that you do not need to include them as options when using `bx d4ic` commands. For example:
+   > You can set your infrastructure API credentials and Docker EE installation URL as environment variables so that you do not have to include them as options when using `bx d4ic` commands. For example:
```

Comment:
I like `need`, as it seems less  vague to people whose first language is not English.